spec/datetimeformat.html

```diff
@@ -514,6 +519,9 @@ <h1>Internal slots</h1>
             <li>month, day</li>
             <li>hour, minute, second</li>
             <li>hour, minute</li>
+            <li>dayPeriod, hour</li>
+            <li>dayPeriod, hour, minute, second</li>
+            <li>dayPeriod, hour, minute</li>
```

Comment:
I don't have a good idea what's the most reasonable minimum subset for day-period display should be. So someone with more background in internalisation/localisation is probably a better reviewer for this specific part.